src/App.tsx

```diff
@@ -10,33 +11,19 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```

Comment:
nit: Consider setting the initial state using a constant or config, for better maintainability. 😊